assets/js/calypsoify.js

```diff
@@ -1,443 +0,0 @@
-			icon = icons.notice;
-		}
-		$( this ).prepend( '<span class="wc-calypso-bridge-notice-icon-wrapper">' + icon + '</span>' );
-	} );
-
-	/**
-	 * Replace dismissal buttons in notices.
-	 */
-	$( document ).ready( function() {
-		$( '.notice-dismiss' ).html( icons.cross );
-	} );
-
-	/**
-	 * Place notice content inside it's own tag.
-	 *
-	 * Used to prevent side by side content in flexbox when multiple paragraphs exist.
-	 */
-	$( 'div.notice, div.error, div.updated, div.warning' ).each( function() {
-		if ( $( this ).children( '.wc-calypso-bridge-notice-content' ).length ) {
-			return;
-		}
-
-		var $noticeContent = $( '<div class="wc-calypso-bridge-notice-content"></div>' );
-		$( this ).find( '.wc-calypso-bridge-notice-icon-wrapper' ).after( $noticeContent );
-		$( this ).find( 'p:not(.submit)' ).appendTo( $noticeContent );
-	} );
-// @todo End
-
-
-	/**
-	 * Show content wrapper.
-	 */
-	$( document ).ready( function() {
-		$( '.wrap' ).fadeIn();
-	} );
-
-	/**
-	 * Move page actions to action header.
-	 */
-	$( document ).ready( function() {
-		var $actions = $( '.page-title-action, .add-new-h2' );
-		if ( ! $actions.is( 'button' ) ) {
-			var buttons = [];
-			$actions.each( function( index, anAction ) {
-				var button = $( '<button/>' ,{
-					class: 'button button-primary',
-					text: anAction.text,
-				} );
-				buttons.push( $( '<a/>' ,{
-			  	class: anAction.className,
-					href: anAction.href
-				} ).append( button ) );
-			} );
-			$( '.woocommerce-layout__activity-panel' ).prepend( buttons );
-			$actions.remove();
-		} else {
-			$actions.prependTo( '.woocommerce-layout__activity-panel' );
-		}
-	} );
-
-	/**
-	 * Move notices on pages with sub navigation.
-	 *
-	 * WP Core moves notices with jQuery so this is needed to move them again since
-	 * we can't control their position.
-	 */
-	$( document ).ready( function() {
-		var $subNavigation = $( '.wrap .subsubsub' );
-		if ( $subNavigation.length ) {
-		$( 'div.notice, div.error, div.updated, div.warning' ).insertAfter( $subNavigation.first() );
-			$( '.jetpack-jitm-message, .jitm-card' ).insertAfter( $subNavigation.first() );
-		}
-	} );
-
-	/**
-	 * Append subnav dropdown for mobile.
-	 */
-	$( document ).ready( function() {
-		$( '#wpbody-content .subsubsub, #wpbody-content .nav-tab-wrapper' ).each( function() {
-			var currentText = $( this ).find( 'a.current, .nav-tab-active' ).text();
-			var $toggle = $( '<div class="nav-tab-toggle"><span class="nav-tab-toggle__current-page">' + currentText + '</span>' + icons.chevronDown + '</div>' );
-			$( this ).wrap( '<div class="nav-tab-container"></div>' );
-			$( this ).before( $toggle );
-		} );
-	} );
-
-	/**
-	 * Append subnav dropdown for mobile.
-	 */
-	$( document ).on( 'click', '.nav-tab-toggle', function() {
-		$( this ).parent().toggleClass( 'is-open' );
-	} );
-
-	/**
-	 * Toggle taxonomy form.
-	 */
-	function toggleTaxonomyForm() {
```

Comment:
I'm not too familiar with this, but could it be function we'd like to keep?

Reply:
This is purely a way to emulate switching pages for taxonomy creation to mimic Calypso's behavior.  I don't think we'll need it going forward.